includes/Modules/Thank_With_Google/Web_Tag.php

```diff
@@ -191,21 +183,21 @@ protected function update_the_content( $content ) {
 	 *
 	 * @since 1.80.0
 	 *
-	 * @return bool True if the current button placement is static.
+	 * @return bool True if the current CTA placement is static.
```

Comment:
`Checks if the current buttton placement is static` should be changed to `Checks if the current CTA placement is static`

Reply:
Updated.